src/components/auth-provider.tsx

```diff
@@ -34,26 +33,34 @@ export const AuthProvider: FC = () => {
     const [, getUser] = useAuthAxios<User>({ method: 'GET', url: '/me' }, { manual: true, autoCatch: false })
     const [, requestLogout] = useAuthAxios({ method: 'DELETE', url: '/users' }, { manual: true })
 
-    // Redirect if already logged
-    useEffect(() => {
-        async function checkLoggedIn() {
-            try {
-                const userResponse = await getUser()
-
-                if (currentUser.id !== userResponse.data.id) {
-                  setCurrentUser(userResponse.data)
+    const [checkingUser, setCheckingUser] = useState(true)
+    useEffect(
```

Comment:
In this case it's on a separate line so all arguments have the same indentation.
The second argument has a comment (disabling eslint to ask the dev to put the dependencies there), so that one needs to be indented like that.